### ISSUE TITLE
add an option to hide password

### ISSUE THREAD

**Issue by axetroy** [AlecAivazis/survey#249]
Entering passwords under Unix is invisible, which is also for security reasons.

**Comment by MarkusFreitag** [comment]
Hi axetroy,
could you please provide a bit more context? There is already a component for password inputs https://github.com/AlecAivazis/survey#password

**Comment by axetroy** [comment]
@MarkusFreitag 

By default, the password input box should be hidden.

This behavior is consistent with the password input of unix

**Comment by MarkusFreitag** [comment]
Ah, you mean that also the "star" placeholder chars should be hidden?

**Comment by axetroy** [comment]
@MarkusFreitag 
yes. This is to prevent the password length from being known by others, thus violently cracking.

**Comment by MarkusFreitag** [comment]
Yeah, I totally understand your point.
@AlecAivazis What do you think?

**Comment by szh** [comment]
This would be solved in #482
